integration-tests/eks/validateResources_test.go

```diff
@@ -23,14 +26,23 @@ import (
 )
 
 const (
-	nameSpace        = "amazon-cloudwatch"
-	addOnName        = "amazon-cloudwatch-observability"
-	agentName        = "cloudwatch-agent"
-	operatorName     = addOnName + "-controller-manager"
-	fluentBitName    = "fluent-bit"
-	dcgmExporterName = "dcgm-exporter"
-	podNameRegex     = "(" + agentName + "|" + operatorName + "|" + fluentBitName + ")-*"
-	serviceNameRegex = agentName + "(-headless|-monitoring)?|" + addOnName + "-webhook-service|" + dcgmExporterName + "-service"
+	nameSpace            = "amazon-cloudwatch"
+	addOnName            = "amazon-cloudwatch-observability"
+	agentName            = "cloudwatch-agent"
+	agentNameWindows     = "cloudwatch-agent-windows"
+	operatorName         = addOnName + "-controller-manager"
+	fluentBitName        = "fluent-bit"
+	fluentBitNameWindows = "fluent-bit-windows"
+	dcgmExporterName     = "dcgm-exporter"
```

Comment:
just to confirm -- dcgm-exporter should *not* be installed on windows nodes. Can you confirm this?

Reply:
yes, this should not be installed on Windows and It is not installed as part of my tests.